src/functions-and-arrays.js

```diff
@@ -1,19 +1,51 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers( num1, num2) {
+  
+  if ( num1 > num2) {
+    return num1;
+  } else if ( num1 < num2 ) {
+return num2;
+  }
+  else {
```

Comment:
Here you could just return one (num1 or num2) because they will have the same value.